src/IntlPhoneInput.js

```diff
@@ -136,7 +140,8 @@ export default class IntlPhoneInput extends React.Component {
       filterText,
       searchIconStyle,
       closeButtonStyle,
-      inputProps
+      inputRef,
```

Comment:
Isn't it necessary to add to prop types?

Reply:
Have added